pkg/controllers/vdb/init_db.go

```diff
@@ -65,6 +70,11 @@ type GenericDatabaseInitializer struct {
 	ConfigurationParams *vtypes.CiMap
 }
 
+type GcpSecret struct {
+	Accesskey string
+	Secretkey string
```

Comment:
Does this need to have specific field names, or could we have used something more generic like `map[string]string` instead?

Reply:
Removed the struct and introduced map